### ISSUE TITLE
Question about coc-rename

### ISSUE THREAD

**Issue by dylanirlbeck** [neoclide/coc.nvim#1777]
Hi! I'm loving coc so far, thought I'm a bit new to Neovim/VimScript. I see in the docs it suggests this remapping for `coc-rename`:

```
" Symbol renaming.
nmap <leader>rn <Plug>(coc-rename)
```

However, when I type `comma+r+n` in normal mode (`comma` is my <leader>), Vim replaces the character under my cursor with `n` instead of prompting me to rename the symbol.

Can someone help me figure out what I'm doing wrong? I doubt this is any sort if issue with `coc`, but thought I'd ask anyway. Thanks in advance!

**Comment by chemzqm** [comment]
:verbose nmap

**Comment by arnavb** [comment]
@dylanirlbeck Hey I was facing the same issue, and in my case it turned out that I hadn't defined my leader early enough in my file so it was using the old value of the leader key, not the new one I set (at the bottom of my file). `:verbose nmap` didn't show the mapping, so I was confused as to why this was happening. I just moved my leader key to the top of my config file.

Either that or another plugin is superseding the mapping.
